taf/utils.py

```diff
@@ -306,6 +310,85 @@ def to_tuf_datetime_format(start_date, interval):
     return datetime_object.isoformat() + "Z"
 
 
+def get_file_details(
+    filepath: str,
+    hash_algorithms: List[str] = ["sha256"],
+    storage_backend: Optional[StorageBackendInterface] = None,
+) -> Tuple[int, Dict[str, str]]:
+
+    # Making sure that the format of 'filepath' is a path string.
+    if not isinstance(filepath, str) or not filepath:
+        raise securesystemslib.exceptions.FormatError(
```

Comment:
We are not catching these anywhere, so let's also replace these `securesystemslib.exceptions.FormatError` and `securesystemslib.exceptions.StorageError` with custom errors